src/server/scripts/Northrend/Ulduar/Ulduar/boss_razorscale.cpp

```diff
@@ -15,15 +15,16 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-/// @todo Harpoon chain from 62505 should not get removed when other chain is applied
-
 #include "ScriptMgr.h"
 #include "GameObject.h"
 #include "GameObjectAI.h"
 #include "InstanceScript.h"
 #include "Map.h"
 #include "MotionMaster.h"
+#include "MoveSplineInit.h"
 #include "ObjectAccessor.h"
+#include "PassiveAI.h"
+#include "PathGenerator.h"
```

Comment:
without it, i got error in:

```cpp
uint32 const pathSize = 11;
G3D::Vector3 const RazorscalePath[pathSize] =
```
"Error (active)		incomplete type is not allowed	scripts_northrend	c:\Users\Bradley\Documents\Trinity\src\server\scripts\Northrend\Ulduar\Ulduar\boss_razorscale.cpp	279	
"